includes/Block/Latest_Stories_Block.php

```diff
@@ -0,0 +1,652 @@
+			$this->get_script_settings()
+		);
+
+		// todo: use register_block_type_from_metadata() once generally available.
+		register_block_type(
+			self::BLOCK_NAME,
+			[
+				'attributes'      => [
+					'align'                => [
+						'type'    => 'string',
+						'default' => 'none',
+					],
+					'viewType'             => [
+						'type'    => 'string',
+						'default' => 'grid',
+					],
+					'numOfStories'         => [
+						'type'    => 'number',
+						'default' => 5,
+					],
+					'numOfColumns'         => [
+						'type'    => 'number',
+						'default' => 2,
+					],
+					'orderByValue'         => [
+						'type'    => 'string',
+						'default' => '',
+					],
+					'isShowingTitle'       => [
+						'type'    => 'boolean',
+						'default' => true,
+					],
+					'isShowingDate'        => [
+						'type'    => 'boolean',
+						'default' => false,
+					],
+					'isShowingAuthor'      => [
+						'type'    => 'boolean',
+						'default' => false,
+					],
+					'isShowingViewAll'     => [
+						'type'    => 'boolean',
+						'default' => false,
+					],
+					'viewAllLinkLabel'     => [
+						'type'    => 'string',
+						'default' => '',
+					],
+					'isShowingStoryPlayer' => [
+						'type'    => 'boolean',
+						'default' => false,
+					],
+					'carouselSettings'     => [
+						'type'    => 'object',
+						'default' => [
+							'autoplay' => false,
+							'delay'    => '',
+							'loop'     => false,
+						],
+					],
+					'authors'              => [
+						'type'    => 'array',
+						'default' => [],
+					],
+					'imageOnRight'         => [
+						'type'    => 'boolean',
+						'default' => false,
+					],
+					'isStyleSquared'       => [
+						'type'    => 'boolean',
+						'default' => false,
+					],
+				],
+				'render_callback' => [ $this, 'render_block' ],
+				'editor_script'   => self::SCRIPT_HANDLE,
+				'editor_style'    => self::SCRIPT_HANDLE,
+				'style'           => self::STYLE_HANDLE,
+			]
+		);
+	}
+
+	/**
+	 * Returns script settings.
+	 *
+	 * @since
+	 *
+	 * @return array Script settings.
+	 */
+	private function get_script_settings() {
+		return [
+			'publicPath' => WEBSTORIES_PLUGIN_DIR_URL . 'assets/js/',
+		];
+	}
+
+	/**
+	 * Initializes class variable $block_attributes.
+	 *
+	 * @since
+	 *
+	 * @param array $block_attributes Array containing block attributes.
+	 *
+	 * @return bool Whether or not block attributes have been initialized with given value.
+	 */
+	protected function initialize_block_attributes( $block_attributes = [] ) {
+		if ( ! empty( $block_attributes ) || ! is_array( $block_attributes ) ) {
+			$this->block_attributes = $block_attributes;
+			return true;
+		}
+		return false;
+	}
+
+	/**
+	 * Renders the block type output for given attributes.
+	 *
+	 * @since
+	 *
+	 * @param array $attributes Block attributes.
+	 *
+	 * @return string Rendered block type output.*
+	 */
+	public function render_block( array $attributes ) {
```

Comment:
now that we have our `Carousel_Rendrer` in place I don't thing we need this. Please update to use appropriate renderer.

Reply:
Will be addressed in `rt/gutenberg-selected-stories` by Edi.